bundler/src/main/java/com/f2prateek/bundler/FragmentBundler.java

```diff
@@ -0,0 +1,37 @@
+package com.f2prateek.bundler;
+
+
+import android.app.Fragment;
+
+/**
+ * Set fragment arguments like a Pro.
+ * @param <F> The fragment to setup.
+ */
+public class FragmentBundler<F extends Fragment> {
```

Comment:
I'm a bit confused by the purpose of this class, looks like all it does is expose the two factory methods that return `FragmentBundleBuilder`. So if clients can never create or access methods on this class, why not just nix it?